docs/getting_started.md

```diff
@@ -228,6 +227,20 @@ Examples:
 ```python
 convert_ply('./test.ply', './test.bin')
 ```
+If you have point clouds in other format (`off`, `obj`, etc.), you can use trimesh to convert them into `ply`.
+```python
+import trimesh
+
+def to_ply(input_path, output_path, original_type):
+    mesh = trimesh.load(input_path, file_type=original_type)  # read file
+    mesh.export(output_path, file_type='ply')  # convert to ply
+```
+Examples:
+
+```python
+to_ply('./test.obj', './test.ply', 'obj')
+```
+More demos about single/multi-modality and indoor/outdoor 3D detection can be found in [demo](../demo).
```

Comment:
Please use the absolute path for the link to demo because users may click it in the web-based documentation.

Reply:
Fix.